thirdparty/github.go

```diff
@@ -263,7 +263,8 @@ func GetCommitEvent(ctx context.Context, oauthToken, repoOwner, repo, githash st
 		"operation": "github api query",
 		"size":      resp.ContentLength,
 		"status":    resp.Status,
-		"commit":    githash,
+		"query":     githash,
+		"commit":    commit,
```

Comment:
do we want to log the entire commit object?

Reply:
I don't think so. I just wanted to be able to validate the query and response SHA.

Reply:
isn't this logging the entire object, though?

Reply:
Oh, sorry, you're right. I thought you were asking the opposite. Fixed.